docs/runtimes/image-build.md

```diff
@@ -162,6 +181,21 @@ project.build_function(
 )
 ```
 
+### Extra arguments
+it is also possible to pass custom arguments and flags to Kaniko.
+The extra_args parameter can be utilized in the build_image, build_function, or during the deployment of the function.
```

Comment:
The `extra_args` parameter can be utilized in `build_image`, `build_function`, or during the deployment of the function.